aas-gui/Frontend/aas-web-gui/src/components/SubmodelElementView.vue

```diff
@@ -131,7 +134,7 @@ export default defineComponent({
     data() {
         return {
             submodelElementData: {} as any, // SubmodelElement Data
-            conceptDescription: {} as any, // ConceptDescription Data
+            conceptDescriptions: {} as any, // Data of Concept Descriptions
```

Comment:
I don't see the relevance of this minimal change in variable names and comments. This introduces many changes with no added benefits. Also has to be tested intensively because many lines of code are affected.